tests/rpc/utils/middleware.test.ts

```diff
@@ -16,19 +18,36 @@ describe('Middleware tests', () => {
     .noShrink();
 
   testProp(
-    'can parse json stream',
+    'can parse json stream and are human readable',
```

Comment:
You should do this as a separate test.

And we want to test all kinds of whitespace, not just newlines. Test for `\t` or spaces and `\v`... etc. Chatgpt all the whitespace relevant characters.

And if you want, use fastcheck to generate those whitespace characters.